dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:latest
```

Comment:
Using "latest" tag for the base image: Using "latest" tag for the base image can lead to unexpected behavior if the image gets updated. It's better to use a specific version of the image to ensure consistency.